quandl/model/model_base.py

```diff
@@ -1,21 +1,26 @@
 class ModelBase(object):
-    def __init__(self, raw_data, **options):
-        self.raw_data = raw_data
+    def __init__(self, code, raw_data=None, **options):
+        self.code = code
+        self._raw_data = raw_data
```

Comment:
can we turn this back to `raw_data`? I think before users could easily access the dictionary of the response via `Object.raw_data`